contents/blog/cheaper-analytics.md

```diff
@@ -0,0 +1,121 @@
+---
+date: 2024-06-21
+title: 'PostHog is now up to 80% cheaper'
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog_is_the_cheapest_e77c4ea4a5.jpg
+author:
+  - james-hawkins
+tags:
+  - PostHog news
+---
+import {ProductScreenshot} from 'components/ProductScreenshot'
+
+export const Pricing = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_ba910c9535.jpg"
+export const PricingDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_dark_6b11387d6c.jpg"
+export const PricingComparison = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_comparison_f0c9e11f40.jpg"
+export const PricingComparisonDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_comparison_dark_ecf7d726c0.jpg"
+
+## tl;dr
+
+* You can now use PostHog's product and web analytics for up to 80% less, making us cheaper than any popular competitive product.
+
+* This is possible because we've introduced new event processing option that allows you to capture events without person information. Previously we processed every event with person information.
+
+* This allows us to reduce our costs for events that do not need person information. We're passing this saving to our customers.
+
+* If you use this option, we are by far the cheapest way to track your product compared to any major analytics competitor.
```

Comment:
```suggestion
* We've split the prices for existing customers, so you'll see an extra line item on your invoice. **Your overall price has not gone up**, it's simply now split between the base event price and the person profile add-on.
```

Restoring @raquelmsmith's suggestion. Raquel, should we mention what the new line item is called?

Reply:
Yep, I edited your suggestion above 👍